src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ChannelPricing.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.channel_pricing.class%" shortName="ChannelPricing">
+        <attribute name="validation_groups">sylius</attribute>
+
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>channel_pricing:read</attribute>
+            </attribute>
+        </attribute>
+
+        <itemOperations>
+            <itemOperation name="shop_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/shop/channel-pricings/{id}</attribute>
+            </itemOperation>
+        </itemOperations>
+
+        <property name="price" writable="false" readable="true" />
```

Comment:
you should add here some `id` to prevent the error from ApiPlatform 2.6

Reply:
And remove collection operations if we are going in this direction.